paddle/gserver/tests/test_LayerGrad.cpp

```diff
@@ -262,6 +262,8 @@ void testConvLayer(const string& type, bool trans, bool useGpu) {
                               config.layerConfig.num_filters());
 
   testLayerGrad(config, "conv", 100, trans, useGpu);
+  // Use small batch_size and useWeight=true to test biasGrad
+  testLayerGrad(config, "conv", 2, trans, useGpu, true, 0.02);
```

Comment:
The reason I set the batchsize to 2 is that the bug in calculating the bpropSharedBias cannot be caught with large batchsize due to averaging out errors.